stores/post_store.jsx

```diff
@@ -156,15 +156,17 @@ class PostStoreClass extends EventEmitter {
     }
 
     getDraft(channelId) {
-        return this.normalizeDraft(BrowserStore.getGlobalItem('draft_' + channelId));
+        // deep clone because many components need to modify the draft
+        return JSON.parse(JSON.stringify(this.normalizeDraft(BrowserStore.getGlobalItem('draft_' + channelId))));
     }
 
     storeCommentDraft(parentPostId, draft) {
         BrowserStore.setGlobalItem('comment_draft_' + parentPostId, draft);
     }
 
     getCommentDraft(parentPostId) {
-        return this.normalizeDraft(BrowserStore.getGlobalItem('comment_draft_' + parentPostId));
+        // deep clone because many components need to modify the draft
+        return JSON.parse(JSON.stringify(this.normalizeDraft(BrowserStore.getGlobalItem('comment_draft_' + parentPostId))));
```

Comment:
why you stringy and parse?

Reply:
more to understand 😄

Reply:
Easy and fast way to deep clone the object https://stackoverflow.com/a/5344074